content/integrate/prometheus-with-redis-enterprise/prometheus-metrics-definitions.md

```diff
@@ -20,262 +20,261 @@ Here are the metrics available to Prometheus:
-| listener_other_res_max | Highest value of rate of other (non read/write) responses on the endpoint (ops/sec) |
-| listener_other_started_res | Number of responses sent from the DB of type "other" |
-| listener_other_started_res_max | Highest value of number of responses sent from the DB of type "other" |
-| listener_read_req | Rate of read requests on the endpoint (ops/sec) |
-| listener_read_req_max | Highest value of rate of read requests on the endpoint (ops/sec) |
-| listener_read_res | Rate of read responses on the endpoint (ops/sec) |
-| listener_read_res_max | Highest value of rate of read responses on the endpoint (ops/sec) |
-| listener_read_started_res | Number of responses sent from the DB of type "read" |
-| listener_read_started_res_max | Highest value of number of responses sent from the DB of type "read" |
-| listener_total_connections_received | Rate of new client connections to the endpoint (connections/sec) |
-| listener_total_connections_received_max | Highest value of rate of new client connections to the endpoint (connections/sec) |
-| listener_total_req | Request rate handled by the endpoint (ops/sec) |
-| listener_total_req_max | Highest value of rate of all requests on the endpoint (ops/sec) |
-| listener_total_res | Rate of all responses on the endpoint (ops/sec) |
-| listener_total_res_max | Highest value of rate of all responses on the endpoint (ops/sec) |
-| listener_total_started_res | Number of responses sent from the DB of all types |
-| listener_total_started_res_max | Highest value of number of responses sent from the DB of all types |
-| listener_write_req | Rate of write requests on the endpoint (ops/sec) |
-| listener_write_req_max | Highest value of rate of write requests on the endpoint (ops/sec) |
-| listener_write_res | Rate of write responses on the endpoint (ops/sec) |
-| listener_write_res_max | Highest value of rate of write responses on the endpoint (ops/sec) |
-| listener_write_started_res | Number of responses sent from the DB of type "write" |
-| listener_write_started_res_max | Highest value of number of responses sent from the DB of type "write" |
+| V1&nbsp;metric | Equivalent V2 PromQL | Description |
+| --------- | :------------------- | :---------- |
+| <span class="break-all">listener_acc_latency</span> | <span class="break-all">N/A</span> | Accumulative latency (sum of the latencies) of all types of commands on the database. For the average latency, divide this value by listener_total_res |
+| <span class="break-all">listener_acc_latency_max</span> | <span class="break-all">N/A</span> | Highest value of accumulative latency of all types of commands on the database |
+| <span class="break-all">listener_acc_other_latency</span> | <span class="break-all">N/A</span> | Accumulative latency (sum of the latencies) of commands that are a type "other" on the database. For the average latency, divide this value by listener_other_res |
```

Comment:
I guess "a type XXX" is OK for these but "of type XXX" is used elsewhere in the table (not a major problem!)